apps/quickbooks_online/actions.py

```
@@ -23,18 +23,59 @@
 logger.level = logging.INFO
 
 
-def update_last_export_details(workspace_id):
+def update_last_export_details(workspace_id: int, skipped_expense_ids: List[int] = None) -> LastExportDetail:
```

Ruff (0.8.2)

Undefined name `List`

(F821)